RHEL/7/input/xccdf/system/software/sudo.xml

```diff
@@ -0,0 +1,65 @@
+<Group id="sudo">
+<title>Sudo</title>
+<description>
+<tt>Sudo</tt>, which stands for "su 'do'", provides the ability to delegate authority
+to certain users, groups of users, or system administrators. When configured for system 
+users and/or groups, <tt>Sudo</tt> can allow a user or group to execute privileged commands
+that normally only <tt>root</tt> is allowed to execute.
+<br/><br/>
+For more information on <tt>Sudo</tt> and addition <tt>Sudo</tt> configuration options, see
+<b>https://www.sudo.ws</b>
+</description>
+
+<Rule id="sudo_remove_nopasswd" severity="medium">
+<title>Remove nopasswd Usage From Sudo</title>
```

Comment:
I would suggest change the title of the rule to something like ```Ensure NOPASSWD Not Used in Sudo Security Policy Modules```. The current title (```Remove nopasswd Usage From Sudo```) suggests the check will potentially remove any occurrence of ```NOPASSWD``` option form Sudo security policy module if some was found (which is not the case, since it will just check ```NOPASSWD``` isn't present in some of the modules).

I would suggest changing rule name to be less confusing. Also wouldn't avoid use of ```NOPASSWD``` directive in capital letters, since the capitalized form is actually the expected one.